cmd/transportRequestUploadSOLMAN.go

```diff
@@ -66,7 +66,7 @@ type iTransportRequestUtils interface {
 type transportRequestUtils struct {
 }
 
-func (g *transportRequestUtils) FindIDInRange(label, from, to string) (string, error) {
+func (*transportRequestUtils) FindIDInRange(label, from, to string) (string, error) {
 	return transportrequest.FindIDInRange(label, from, to)
```

Comment:
Wait, so `transportrequest` is somehow magically of type `*transportRequestUtils`?

Reply:
_transportRequestUtils_ implements _iTransportRequestUtils_ containing a redirection to the real implementation of _FindIDInRange_ which is done in the package _transportrequest_

Reply:
No, transportrequest is the imported package. So there is no need for FindIDInRange to have a pointer receiver, or is there?

Reply:
Is it that you like the structure of calling `trUtils.FindIDInRange` instead of `FindIDInRange` in the below functions `getTransportRequestID` and `getChangeDocumentID`?

Reply:
That is the way to implement the mocking layer, as far as I understood. See the according _test.go file.
It looks quite complex - actually, it is - but by this we have a clear setup.

Reply:
i found the discussion here: https://github.com/SAP/jenkins-library/pull/1913#issuecomment-704801684